### ISSUE TITLE
when filling non textual fields, typing "enter" should not open "completude" popup

### ISSUE THREAD

**Issue by mgerbaux** [c2corg/v6_ui#1528]
It is the case for outings or routes, but not for articles.
It is especially destabilizing for auto-completion fields on which newbies are tempted tempted to validate their keyboarding

**Comment by brunobesson** [comment]
Could you more explicit? I understand the global meaning of the issue, but could you point to both a working and a non-working examples?

**Comment by mgerbaux** [comment]
> Could you more explicit?
hardly…
Try it ? 
![screenshot at 2017-02-24 13 02 57](https://cloud.githubusercontent.com/assets/22116605/23303001/be67711c-fa91-11e6-83c6-066ba0030742.png)
(I have typed the whole summit name then enter before autocompletion propose anything)

**Comment by asaunier** [comment]
Perhaps related to this issue https://github.com/c2corg/v6_ui/issues/1293 ?

**Comment by brunobesson** [comment]
That hitting enter submits the form in some cases is not really a problem, this is a standard web feature. What I don't understand, is why the completion modal shows up while some fields are invalid. Can you submit the form when clicking "Save" in the dialog then? I guess not.

Thus, @asaunier guess might be the correct one.

**Comment by asaunier** [comment]
> It is the case for outings or routes, but not for articles

Indeed the behaviour is observed with all document types except articles and books, which are the only documents with no map. I have tried to remove the maps in the editing forms of waypoints and outings: the problem no longer occurs.

Then I am pretty sure this issue is a duplicate of https://github.com/c2corg/v6_ui/issues/1293 and could be closed. I have ported the behaviour described here to https://github.com/c2corg/v6_ui/issues/1293#issuecomment-302407447
